[MULTIARCH-3485] Single Node Cluster Support (SNO) for P (#65166)
openshift/openshift-docs: modules/setting-up-bastion-for-sno.adoc

// This module is included in the following assemblies:
//
// installing_sno/install-sno-installing-sno.adoc

:_content-type: PROCEDURE
[id="setting-up-bastion-for-sno_{context}"]
= Setting up basion for {sno} with {ibmpowerProductName}

Prior to installing {sno} on {ibmpowerProductName}, you must set up bastion. Setting up a bastion server for {sno} on {ibmpowerProductName} requires the configuration of the following services: 

> * PXE is used for the {sno} cluster installation. It requires the following services to be configured and run:

Reviewer: ```suggestion
* PXE is used for the {sno} cluster installation. PXE requires the following services to be configured and run:
```

> ** DNS to define api, api-int and *.apps

Reviewer: ```suggestion
** DNS to define api, api-int, and *.apps
```

> ** DHCP service to enable PXE and assign IP to {sno} node

Reviewer: is it assigning IP addresses? or an IP address? I assume it should be: 

```suggestion
** DHCP service to enable PXE and assign an IP address to {sno} node
```

> ** HTTP to provide ignition and RHCOS rootfs image

Reviewer: ```suggestion
** HTTP to provide ignition and {op-system} rootfs image
```

** TFTP to enable PXE
* You must install `dnsmasq` to support DNS, DHCP and PXE, httpd for HTTP.

> Use the following procedure to configure a bastion server that meets these requirements.

Reviewer: Need a space here between lines 16 and 17. Content is on the same line as the previous bullet point. See rendering for what I'm talking about if you need to.

Author: Ack. Updated the PR. Thanks.
